database/index.js

```diff
@@ -1,8 +1,35 @@
 const mysql = require('mysql');
 const mysqlConfig = require('./config.js');
-
 const connection = mysql.createConnection(mysqlConfig);
 
+const createWatchInfo = (watchId, watchName, options, callback) => {
+  let strapIds = [1, 2, 3, 4, 5];
+  let strap_id = strapIds[(Math.random().toFixed(1).split('.')[1] % 5)];
+  const queryStringWatchInfo = 
+  `INSERT INTO watches (wid, watch_name, series, size, watch_price) 
+  VALUES (
+    "${watchId}", "${watchName}", 
+    "${options.series}", "${options.size}",
+    "${options.watch_price}"
+  );`;
+  const queryStringStrapOptionsInfo = `INSERT INTO strap_options (watch_id, strap_id) VALUES ("${(watchId)}", "${(strap_id)}");`
+  connection.query(queryStringWatchInfo, (watchErr, watchRes) => {
+    if (watchErr) {
+      throw watchErr;
+    } else {
+      const resultArray = [watchRes];
+      connection.query(queryStringStrapOptionsInfo, (strapOptionsErr, strapOptionsRes) => {
+        if (strapOptionsErr) {
+          throw strapOptionsErr;
+        } else {
+          resultArray.push(strapOptionsRes);
+          callback(null, resultArray);
```

Comment:
You may want to change this to result instead of resultArray, since I believe you won't receive an actual data-based response for any query except 'SELECT' for your get request.